requirements.txt

```diff
@@ -54,3 +54,5 @@ websocket-client==1.7.0 ; python_version >= "3.10"
 werkzeug==3.0.2 ; python_version >= "3.10"
 wtforms==3.1.2 ; python_version >= "3.10"
 yarl==1.9.4 ; python_version >= "3.10"
+PyJWT
```

Comment:
you can run `make deps-update` and it should add the version number in correct format

Reply:
done. Thanks